### PR TITLE
Added Teensy stager

### PR DESCRIPTION
**Pull request by matterpreter**
Added a Teensy stager that generates a Teensy script that runs a
one-liner stage0 launcher.

### CODE REVIEW

**Comment by killswitch-GUI** [comment]
@matterpreter Thanks for the pull! We will review (Cant test Teensy ATM). May have to pick one up :)

**Comment by matterpreter** [comment]
Awesome! I'm using a 3.2, but it should work on all versions.

File: lib/stagers/teensy.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        self.options = {
+            # format:
+            #   value_name : {description, required, default_value}
+            'Listener' : {
+                'Description'   :   'Listener to generate stager for.',
+                'Required'      :   True,
+                'Value'         :   ''
+            },
+            'OutFile' : {
+                'Description'   :   'File to output AppleScript to, otherwise displayed on the screen.',
```
Comment (killswitch-GUI) [inline lib/stagers/teensy.py]: Please update description to Teensy.

**Comment by killswitch-GUI** [comment]
Please update that one line note, all else looks good!

**Comment by matterpreter** [comment]
Good catch! Fixed in 3f83c5c.

**Comment by killswitch-GUI** [comment]
Thanks for the pull!

**Pull request merged**